src/util/apiUtils.js

```diff
@@ -26,6 +27,13 @@ const MOCK_RESPONSE_OK = {  // minimal representation of http.IncomingMessage
 	},
 };
 
+export const createCookieJar = apiUrl => {
+	if (url.parse(apiUrl).pathname === '/sessions') {
```

Comment:
I was debating between putting the check in `buildRequestArgs` vs putting it here, but I decided to put it here because it keeps the logic in `buildRequestArgs` simple and declarative, and gives `createCookieJar` sole responsibility for creating a cookie jar value - if someone were to ask "where is the logic related to assigning a cookie jar to a request?", I think `createCookieJar` makes more sense than `buildRequestArgs`.

There's actually still more logic in `buildRequestArgs` than I'd like, particularly the `switch` statement.